From 052c5d53d56f52cba95a569b3f2648b7cd647f1e Mon Sep 17 00:00:00 2001
From: Nathaniel McCallum <npmccallum@redhat.com>
Date: Thu, 9 Aug 2018 15:07:17 -0400
Subject: [PATCH] Specify LUKSv1 during luksFormat

This fixes tests on builds of cryptsetup which default to LUKSv2.
---
 test-luksmeta | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test-luksmeta b/test-luksmeta
index fd17ead..f1e8b2e 100755
--- a/test-luksmeta
+++ b/test-luksmeta
@@ -11,7 +11,7 @@ function onexit() {
 trap 'onexit' EXIT
 
 truncate -s 4M $tmp
-echo -n foo | cryptsetup luksFormat $tmp -
+echo -n foo | cryptsetup luksFormat --type luks1 $tmp -
 
 ! ./luksmeta test -d $tmp
 
-- 
2.17.1

